src/wmnds/components/form-elements/file-upload/_properties.njk.json

```
@@ -0,0 +1,83 @@
+{
+  "wmndsFileUploaderProps": [
+    {
+      "name": "name",
+      "type": "string",
+      "description": "<strong>Required.</strong>. The name of the input, which is submitted with the form data."
+    },
+    {
+      "name": "id",
+      "type": "string",
+      "description": "<strong>Required.</strong> Must be unique. The id of the input"
+    },
+    {
+      "name": "value",
+      "type": "string",
+      "description": "Optional initial value of the input"
+    },
+    {
+      "name": "classes",
+      "type": "string",
+      "description": "Classes to add to the file upload component."
+    },
+    {
+      "name": "required",
+      "type": "boolean",
+      "description": "If true, file upload is required/must be filled out. Defaults to <code class='wmnds-website-inline-code'>false</code>."
+    },
+    {
+      "name": "disabled",
+      "type": "boolean",
+      "description": "If true, file upload should be disabled. Defaults to <code class='wmnds-website-inline-code'>false</code>."
+    },
+    {
+      "name": "fileSelected",
+      "type": "boolean",
+      "description": "If true, it'll simulate that the file was already selected. Defaults to <code class='wmnds-website-inline-code'>false</code>."
+    },
+    {
+      "name": "error",
+      "type": "boolean",
+      "description": "If true, errorMessage will show. Defaults to <code class='wmnds-website-inline-code'>false</code>."
+    },
+    {
+      "name": "errorMessage",
+      "type": "object",
+      "description": "Options for the error message component. See errorMessage.",
+      "arrayOptions": [
+        {
+          "name": "id",
+          "type": "string",
+          "description": "Id attribute to add to the error message span tag."
+        },
+        {
+          "name": "contentText",
+          "type": "string",
+          "description": "<strong>Required.</strong> Text to use within the error message. If <code class='wmnds-website-inline-code'>contentHTML</code> is supplied, this is ignored."
+        },
+        {
+          "name": "contentHTML",
+          "type": "string",
+          "description": "<strong>Required.</strong> HTML to use within the error message"
+        },
+        {
+          "name": "classes",
+          "type": "string",
+          "description": "Classes to add to the error message span tag."
+        }
+      ]
+    },
+    {
+      "name": "formGroup",
+      "type": "Object",
+      "description": "Options for the form-group wrapper See formGroup below.",
+      "arrayOptions": [
+        {
+          "name": "classes",
+          "type": "string",
+          "description": "Classes to add to the form group (e.g. to show error state for the whole group)"
```

```suggestion
          "description": "Classes to add to the form group"
```
could be a little confusing as the error message already adds the error class to the form group

---

completely right! 👍